doc/de/groupsandpages.md

```diff
@@ -1,22 +1,22 @@
-Gruppen und Seiten 
+Gruppen und Seiten
 ==========
 
 * [Zur Startseite der Hilfe](help)
 
-Hier siehst du das globale Verzeichnis. 
-Wenn du dich mal verirrt hast, kannst du <a href = "help/groupsandpages">diesen Link klicken</a> und wieder hierher kommen. 
+Hier siehst du das globale Verzeichnis.
+Wenn du dich mal verirrt hast, kannst du <a href = "help/groupsandpages">diesen Link klicken</a> und wieder hierher kommen.
 
-Auf dieser Seite findest du eine Zusammenstellung von Gruppen, Foren und bekannten Seiten. 
-Gruppen sind keine realen Personen. 
-Sich mit diesen zu verbinden ist, als wenn man jemanden auf Facebook "liked" ("gefällt mir") oder wenn man sich in einem Forum anmeldet. 
+Auf dieser Seite findest du eine Zusammenstellung von Gruppen, Gruppen und bekannten Seiten.
```

Comment:
I guess that we haven't yet decided on how name circles in German. Possibly we should name them "Circles" just as a possibility to distinct them from the groups.

Reply:
This page was also confusing in English, I ended up wrongly replacing a "group" (page) by "circle" in the first PR. I've fixed the mistake in this PR but not on the German translation for obvious reasons.

Which means neither instance of "Gruppen" in this file needs to be replaced by "Circles", it turns out we've been mentioning forums as "groups" for a while here.

Reply:
We could use the English word "Circle" in the German translation. Then we can use "Gruppen" without a problem.

Reply:
This is what I went for. This page needs a touch-up like the English one did since the original "group" term in either language didn't mean "contact group".

Reply:
I've edited the German version to be in line with the English version.